MAUIsland/Features/Gallery/MAUI/GridLayout/Pages/GridLayoutPage.xaml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<app:BasePage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:app="clr-namespace:MAUIsland"
+             
+             x:Class="MAUIsland.GridLayoutPage"
+             x:DataType="app:GridLayoutPageViewModel"
+             
+             
+             Title="GridLayoutPage">
+    <VerticalStackLayout>
```

Comment:
Update this to our new standard include these: 

1. Control Info
2. Properties List Header
3. Properties List Footer
4. Properties Item Source